lib/esshorten.js

```diff
@@ -82,9 +82,17 @@
         }
     }
 
-    const run = (scope, options) => {
+    const run = (scope, options, cb) => {
         let generator = new NameGenerator(scope, options);
 
+        function shouldRename(name) {
+            var should = true;
```

Comment:
There's no need to use a flag here. Instead of performing the test for the existence of the predicate every time it is called, do it once to choose whether to use the predicate or a function that always returns true.